ProjectName/src/screens/PaymentDetails/PaymentDetailsForm/components/customAmountInput/CustomAmountInput.tsx

```diff
@@ -0,0 +1,53 @@
+import React, {FC} from 'react';
+import {Controller} from 'react-hook-form';
+import {View, Text, TextInput, Image} from 'react-native';
+
+import {DECIMAL_REGEX} from './custom-amount-input.constants';
+import {styles} from './custom-amount-input.styles';
+import {CustomMessageInputTypes} from '../customMessageInput/custom-message-input.types';
+
+const CustomAmountInput: FC<CustomMessageInputTypes> = ({
+  control,
+  name,
+  title,
+  placeholder,
+}) => {
+  return (
+    <View>
```

Comment:
I guess we might not need to use extra `View` here. Regular Fragment is just enough `<></>`